src/ui/utils/telemetry.ts

```diff
@@ -66,12 +67,13 @@ export async function sendTelemetryEvent(event: string, tags: any = {}) {
     if (skipTelemetry()) {
       return;
     }
+    tags["user"] = telemetryUser;
     const response = await fetch("https://telemetry.replay.io/", {
       method: "POST",
       headers: {
         "Content-Type": "application/json",
       },
-      body: JSON.stringify({ event, ...tags, user: telemetryUser }),
+      body: JSON.stringify({ event, ...tags }),
```

Comment:
hmm, why do we need to change this? In general, i think it is a bit cleaner to set the user in the new object as opposed to mutating the tags

Reply:
Yeah I'll revert this change.